TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -30,22 +30,28 @@ class RouteDetailContentViewController: UIViewController {
     /// Number of seconds to wait before auto-refreshing live tracking network call call, timed with live indicator
     var liveTrackingNetworkRefreshRate: Double = LiveIndicator.interval * 1.0
     var liveTrackingNetworkTimer: Timer?
-    private var locationManager = CLLocationManager()
     var mapView: GMSMapView!
-    private let networking: Networking = URLSession.shared.request
+    private var locationManager = CLLocationManager()
```

Comment:
i think these properties are in alphabeticalc order. not a huge deal but might as well stay consistent :)